generators/GitHubIssueGenerator.py

```diff
@@ -230,7 +241,15 @@ def open_github_issue(self):
                 except UnknownObjectException as ex:
                     print(f"Couldn't find GitHub Tag {tag}, skipping and continuing.", file=sys.stderr, flush=False)
                     pass
-            _issue = repo.create_issue(self.generate_issue_title(), body=_message, labels=_github_tags_objects)
+            _assignees=[]
+            for tag in _tags:
+                try:
+                    if tag in self.assigneelist:
+                        _assignees.append(org.get_members(self.assigneelist[tag]))
```

Comment:
Does this look right in terms of obtaining a github user object?

Reply:
`org.get_members` will return a list of all users in the org (no matter what the argument is).

Reply:
How about my latest push?